src/picongpu/submit/titan-ornl/picongpu.profile.example

```diff
@@ -9,6 +10,8 @@ module swap PrgEnv-pgi PrgEnv-gnu
 #   (CMake likes to unwrap the Cray wrappers)
 export CC="`which cc` -target=compute_node"
```

Comment:
Note from ORNL support:
using
```bash
cmake -DCMAKE_CXX_COMPILER=`which CC` [...]
```
might result in the same behavior (CMake should not unwrap the compiler-wrappers to their underlying compiler such as gcc).